FLEDGE.md

```diff
@@ -509,7 +509,11 @@ reportResult(auctionConfig, browserSignals, directFromSellerSignals) {
 The arguments to this function are:
 
 *   auctionConfig: The auction configuration object passed to `navigator.runAdAuction()`
-*   browserSignals: An object constructed by the browser, containing information it knows about what happened in the auction. `topLevelSeller`, `topLevelSellerSignals`, and `modifiedBid` are only present for component auctions, while `componentSeller` is only present for top-level auctions when the winner came from a component auction. `modifiedBid` is the bid value a component auction's `scoreAd()` script passes to the top-level auction. `topLevelSellerSignals` is the output of the top-level seller's ReportResult() method:
+*   browserSignals: An object constructed by the browser, containing information it knows about what happened in the auction.
+    *   `topLevelSeller`, `topLevelSellerSignals`, and `modifiedBid` are only present for component auctions, while `componentSeller` is only present for top-level auctions when the winner came from a component auction.
+    *   `modifiedBid` is the bid value a component auction's `scoreAd()` script passes to the top-level auction.
+    *   `topLevelSellerSignals` is the output of the top-level seller's ReportResult() method.
+    *   `highestScoringOtherBid` is the value of a bid with the second highest score in the auction. It may be greater than `bid` since it's a bid instead of a score, and a higher bid value may get a lower score. Rejected bids are excluded when calculating this signal. If there was only one bid, it will be 0. In the case of a tie, it will be randomly chosen from all bids with the second highest score, excluding the winning bid if the winning bid had the same score. A component seller's reportWin() function will be passed a bid with the second highest score in the component auction, not the top-level auction. It is not reported to top-level sellers in a multi-SSP case because we expect a top-level auction in this case to be first-price auction only:
```

Comment:
> It is not reported to top-level sellers in a multi-SSP case because we expect a top-level auction in this case to be first-price auction only

The motivation for knowing the second-highest bid can extend beyond determining the cost of an impression in a second-price auction. The second-highest bid information can also be quite valuable and important in first-price auctions for training pricing models that help buyers bid optimally and avoid paying more than necessary to aquire a given impression.

Is `highestScoringOtherBid` intended to support the latter use case?

Reply:
> The second-highest bid information can also be quite valuable and important in first-price auctions for training pricing models that help buyers bid optimally and avoid paying more than necessary to aquire a given impression.

`highestScoringOtherBid` is available to all buyers.  The line you're quoting only refers to top-level sellers not receiving this signal.